ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/deploy_container/scripts/dockerCPU.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+# Usage - Script takes one argument, the number of cpus for the docker container
+# ie bash dockerCPU.sh 4
+# Argument cannot be 0 or greater than the number of CPUs on the machine
+# If /var/log/docker_cpu does not exist, run: echo 0 > /var/log/docker_cpu
+
+set -eu
+
+cpu_limit=$1
+start_cpu=$(tail -n 1 /var/log/docker_cpu)
```

Comment:
Since it is a machine configuration thing can we change this to be a location under `/etc` e.g. `/etc/adoptium_docker_cpu`? Putting it in a log directory doesn't feel right (It could also be somewhere like `/etc/adoptium/docker_cpu` or `/usr/local/etc/adoptium_docker_cpu`